main.js

```diff
@@ -1,9 +1,87 @@
-document.addEventListener("DOMContentLoaded", function() {
-    // Code to be executed when the DOM is ready
-    let videoButton = document.getElementById('button-video');
-    let videoBanner = document.getElementById('banner-video');
-
-    videoButton.onclick = function () {
-        videoBanner.play();
-    };
-}); 
+document.addEventListener("DOMContentLoaded", function () {
+  // Code to be executed when the DOM is ready
+  let videoButton = document.getElementById("button-video");
+  let videoBanner = document.getElementById("banner-video");
+
+  videoButton.onclick = function () {
+    if (videoBanner.paused) {
+      videoBanner.play();
+    } else {
+      videoBanner.pause();
+    }
+  };
+
+  let appointmentForm = document.getElementById("appointmentForm");
+  appointmentForm.addEventListener("submit", validationForm);
+  function validationForm (e) {
+    e.preventDefault();
+    let fullName = document.querySelector("[name='fullname']").value;
+    let email = document.querySelector("[name='email']").value;
+    let message = document.querySelector("[name='message']").value;
+    let date = document.querySelector("[name='date']").value;
+    let time = document.querySelector("[name='time']").value;
+
+    let nameError = document.getElementById("nameError");
+    let emailError = document.getElementById("emailError");
+    let messageError = document.getElementById("messageError");
+    let dateError = document.getElementById("dateError");
+    let timeError = document.getElementById("timeError");
+
+    nameError.innerHTML = "";
+    emailError.innerHTML = "";
+    messageError.innerHTML = "";
+    dateError.innerHTML = "";
+    timeError.innerHTML = "";
+
+    if (fullName.split(" ").filter(Boolean).length < 2) {
+      nameError.innerHTML = "Full Name should content 2 words";
```

Comment:
Full name should contain minimum 2 letters, not words

Reply:
I know, but I decided to make validation for two words, cause it's fullname and need to content first name and last name - 2 words)))